concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limiter/BlockingLimiter.java

```diff
@@ -52,40 +50,35 @@ private BlockingLimiter(Limiter<ContextT> limiter) {
     
     private void unblock() {
         synchronized (lock) {
-            if (blocked) {
```

Comment:
worth adding some class-level javadoc to indicate that this class is thread-safe?

Reply:
Ultimately if you're going to block, you need another thread to wake you up, so I think it can be reasonably assumed?